### PR TITLE
ensure config exists

### LINKED ISSUE #35: /etc/gitlab-runner/config.toml must exist
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 5.5.8
- Ruby: 2.5.1
- Distribution: Ubuntu Bionic
- Module version: https://github.com/voxpupuli/puppet-gitlab_ci_runner/commit/3029b6eea36ef082410f3352ffcbc040cedcc2f0

## How to reproduce (e.g Puppet code you use)

The existing module.

## What are you seeing

I was attempting a manual upgrade to gitlab runner along with needing to reregister the runner against gitlab, the easiest way at the seemed to be to just delete /etc/gitlab-runner/config.toml

When I then ran puppet the module failed because it tried to ensure a file_in_line on the config file with out it existing. 

## What behavior did you expect instead

I expected the module to:
1. have all file_line resources require that config.toml exists.
and
1. create a blank file if it doesn't exist so the configuration can be populated

## Output log
`/opt/puppetlabs/puppet/bin/puppet agent --server server1 --verbose --test`
....
```
Error: /Stage[main]/Gitlab_ci_runner/File_line[gitlab-runner-concurrent]: Could not evaluate: No such file or directory @ rb_sysopen - /etc/gitlab-runner/config.toml
Notice: /Stage[main]/Gitlab_ci_runner/Exec[gitlab-runner-restart]: Dependency File_line[gitlab-runner-concurrent] has failures: true
Warning: /Stage[main]/Gitlab_ci_runner/Exec[gitlab-runner-restart]: Skipping because of failed dependencies
```

`touch /etc/gitlab-runner/config.toml`
`/opt/puppetlabs/puppet/bin/puppet agent --server server1 --verbose --test`
...
```
Notice: /Stage[main]/Gitlab_ci_runner/File_line[gitlab-runner-concurrent]/ensure: created
Info: /Stage[main]/Gitlab_ci_runner/File_line[gitlab-runner-concurrent]: Scheduling refresh of Exec[gitlab-runner-restart]
Notice: /Stage[main]/Gitlab_ci_runner/Exec[gitlab-runner-restart]: Triggered 'refresh' from 1 event
Notice: Applied catalog in 2.89 seconds
```
## Any additional information you'd like to impart
Above output has been trimmed. As you can see simply touching the file allowed the module to work as expected.

### PR DESCRIPTION
**Pull request by thde**
#### Pull Request (PR) description
This PR ensures that the config file exists.

#### This Pull Request (PR) fixes the following issues
Fixes #35

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -101,9 +101,16 @@
     ensure => $package_ensure,
   }
 
+  $config_path = '/etc/gitlab-runner/config.toml'
```
Comment (LongLiveCHIEF) [inline manifests/init.pp]: Let's make this a module parameter so users can change this value when calling `class {'gitlab_ci_runner'`

Comment (thde) [reply manifests/init.pp]: I added the variable as parameter. There is also a test for this in place.

**Comment by thde** [comment]
I updated the PR

**Comment by LongLiveCHIEF** [comment]
Can you amend the examples in the README to include the new param? It will fit the style of the existing "docs" (heh)  for this module without making you feel like you're responsible for owning this functionality.

I will merge once that's done.

**Comment by thde** [comment]
Sure @LongLiveCHIEF. I added the line in the usage section.

**Pull request merged**